src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -59,6 +60,11 @@ internal sealed class InProcRemoteHostClient : RemoteHostClient
 
         public AssetStorage AssetStorage => _inprocServices.AssetStorage;
 
+        public void RegisterService(string name, Func<Stream, IServiceProvider, ServiceHubServiceBase> serviceCreator)
```

Comment:
can you explain the overall change you're making here.  it's not clear what this is all for.